src/brasil/gov/agenda/browser/agenda.py

```diff
@@ -22,6 +24,9 @@ def update(self):
                                       name='plone_tools')
         context_state = getMultiAdapter((self.context, self.request),
                                         name=u'plone_context_state')
+        portal_state = getMultiAdapter((self.context, self.request),
```

Comment:
follow conventions for [breaking lines](http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#breaking-lines) here and all over the code